wp-content/civi-extensions/goonjcustom/Civi/UrbanPlannedVisitService.php

```diff
@@ -83,4 +85,136 @@ private static function getOutcomeEmailHtml($coordinatingGoonjPOCName, $visitId)
     return $html;
   }
 
+  /**
+   * This hook is called after a db write on entities.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The name of the object.
+   * @param int $objectId
+   *   The unique identifier for the object.
+   * @param object $objectRef
+   *   The reference to the object.
+   */
+  public static function updateVisitStatusAfterAuth(string $op, string $objectName, $objectId, &$objectRef) {
+    $visitStatusDetails = self::checkVisitStatusAndIds($objectName, $objectId, $objectRef);
+
+    if (!$visitStatusDetails) {
+      return;
+    }
+
+    $newVisitStatus = $visitStatusDetails['newVisitStatus'];
+    $currentVisitStatus = $visitStatusDetails['currentVisitStatus'];
+
+    if ($currentVisitStatus !== $newVisitStatus) {
+      if ($newVisitStatus === 'completed') {
+        $visitId = $objectRef['id'] ?? NULL;
+        if ($visitId === NULL) {
+          return;
+        }
+
+        $visitFeedbackSent = EckEntity::get('Institution_Visit', TRUE)
+          ->addSelect('Visit_Feedback.Feedback_Email_Sent')
+          ->addWhere('id', '=', $visitId)
+          ->execute()->single();
+
+        $isVisitFeedbackSent = $visitFeedbackSent['Visit_Feedback.Feedback_Email_Sent'];
+
+        if ($isVisitFeedbackSent !== NULL) {
+          return;
+        }
+
+        $externalCoordinatingPocId = $objectRef['Urban_Planned_Visit.External_Coordinating_PoC'] ?? '';
+
+        $externalCoordinatingGoonjPOC = Contact::get(FALSE)
+          ->addSelect('email.email', 'display_name')
+          ->addJoin('Email AS email', 'LEFT')
+          ->addWhere('id', '=', $externalCoordinatingPocId)
+          ->execute()->single();
+
+        $externalCoordinatingGoonjPOCEmail = $externalCoordinatingGoonjPOC['email.email'];
+        $externalCoordinatingGoonjPOCName = $externalCoordinatingGoonjPOC['display_name'];
```

Comment:
can you fix the naming here ?
changed this `externalCoordinatingGoonjPOCEmail`
to `externalCoordinatingGoonjPocEmail`

and Please check other places here

Reply:
sure